packages/ariakit-react-core/src/tab/tab.tsx

```diff
@@ -84,7 +84,7 @@ export const useTab = createHook<TagName, TabOptions>(function useTab({
   const selected = store.useState((state) => !!id && state.selectedId === id);
   const hasActiveItem = store.useState((state) => !!store.item(state.activeId));
   const canRegisterComposedItem = isActive || (selected && !hasActiveItem);
-  const accessibleWhenDisabled = selected || props.accessibleWhenDisabled;
+  const accessibleWhenDisabled = selected || ( props.accessibleWhenDisabled ?? true );
```

Comment:
Looks like there are formatting checks

```suggestion
  const accessibleWhenDisabled =
    selected || (props.accessibleWhenDisabled ?? true);
```

Reply:
Oops, sorry, saw your message late, was setting the codebase up locally and fixed it myself.